java-test-samples/apigw-lambda-ddb/LoadTests/load-test-dynamic-data.yml

```diff
@@ -0,0 +1,17 @@
+config:
+  ## Target will be your api endpoint base url
+  target: "<<APIGW ENDPOINT>>/<<STAGE>>"
```

Comment:
SAM template output provides the `PaymentsEndpoint` including the stage name (`/dev`) and resource path (`/ticket`). That will mislead as to how users replace that url here. Wouldn't it be easier to [have a wrapper script](https://github.com/aws-samples/serverless-java-frameworks-samples/blob/main/springboot/load-test/run-load-test.sh) which gets the SAM template output and feed the url here instead? then update readme instructions to run the script?

Reply:
Added script to fetch api endpoint from the cloudformation stack.